### ISSUE TITLE
Error in documentation ?

### ISSUE THREAD

**Issue by devl00p** [Fahrj/reverse-ssh#17]
About the reverse shell scenario there is the following instruction in the README:

```
# On attacker (get ready to catch the incoming request;
# can be omitted if you already have an ssh daemon running, e.g. OpenSSH)
# NOTE: LPORT of 8888 collides with incoming connections; use the flag `-b 8889` or similar on the victim in that case
attacker$ ./reverse-ssh -v -l -p <LPORT>
```
As I understand it, this command is useful if I want to receive a shell from the victim without having a SSH server on my side, right ?

f I try to use it:

```
$ ./reverse-sshx64 -v -l -p 9999
2021/12/12 15:59:06 Dialling home via ssh to 9999:22
2021/12/12 15:59:06 dial tcp: lookup 9999: no such host
```
It looks like it is not listening but is trying to connect to another host. Also the LPORT parameter given is the documentation seems to be used as a HOST instead.

It may be an issue in the documentation or it is not clear to me. I already used it in bind mode and it was great, nice tool  :)

**Comment by Fahrj** [comment]
Hi @devl00p 

you are completely right with your understanding of the scenario and that letting reverse-ssh listen on your (the attacker's) side is to receive the shell coming from the victim.

In your case, unfortunately, you stumbled upon a **mismatch in the current Readme (v1.2.0-dev) and the latest release (v1.1.0)**.

More precisely, the way how `-l` is handled changed: In version 1.1.0 `-l` [required an extra argument that specified the listening address as in HOST:PORT](https://github.com/Fahrj/reverse-ssh/blob/v1.1.0/Readme.md#simple-reverse-shell-scenario). In the current version, `-l` is only a boolean flag that enforces listening mode and the listening port can be specified by `-p` (Host was dropped, since reverse-ssh is now listening on all interfaces/addresses).

For you this means that `./reverse-sshx64 -v -l -p 9999` actually provides the flag `-l` with argument `"-p"`, and `9999` is then a stand-alone argument interpreted as the host address.

**tl;dr:** Use `./reverse-sshx64 -v -l  :9999` with **version v1.1.0**.

**Comment by Fahrj** [comment]
Alternatively, check out [release v1.2.0](https://github.com/Fahrj/reverse-ssh/releases/tag/v1.2.0)

**Comment by devl00p** [comment]
Thank you, I love the -l -p just like Netcat or Ncat. I moved to the new release
